components/badge/badge.css

```diff
@@ -0,0 +1,23 @@
+.badge {
+  width: 1.5rem;
+  height: 1.5rem;
+  border-radius: 50%;
+  position: absolute;
+  right: 10%;
+}
+.badge-online {
+  background-color: green;
```

Comment:
Try to use variables as much as possible, especially with colors

Reply:
Sure I'll change absolute colors to variables